### PR TITLE
Added missing files to lint script and fixed lint errors

### PR DESCRIPTION
**Pull request by sedge**

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
lgtm :+1:

File: authentication/scheme.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * Authentication scheme used to authenticate requests to publish.webmaker.org
 */
 
-var Boom = require('boom');
+// var Boom = require('boom');
```
Comment (ashleygwilliams) [inline authentication/scheme.js]: why comment out?

Comment (sedge) [inline authentication/scheme.js]: Right? This file uses `boom` in its example code below, so I figured I'd leave this to make it clear that it's a dependency.

Comment (sedge) [inline authentication/scheme.js]: Oh, and jshint complains if I don't cuz the variable isn't used anywhere.

**Pull request merged**